lib/pure/collections/deques.nim

```diff
@@ -52,46 +51,69 @@
 ## * `channels module <channels.html>`_ for inter-thread communication
 
 
-import math
+import math, hashes
 
 type
   Deque*[T] = object
     ## A double-ended queue backed with a ringed seq buffer.
     ##
-    ## To initialize an empty deque use `initDeque proc <#initDeque,int>`_.
+    ## To initialize an empty deque with a given capacity use
+    ## `initDeque proc <#initDeque,int>`_.
     data: seq[T]
     head, tail, count, mask: int
 
 const
-  defaultInitialSize* = 4
+  dequeDefaultInitialCapacity* {.intdefine.} = 4
```

Comment:
all new defines need package prefix to avoid clashes and be consistent => `nimDequeDefaultInitialCapacity`
see https://nim-lang.github.io/Nim/contributing.html#best-practices